### PR TITLE
Suggest Full

### PR DESCRIPTION
**Pull request by Huan200799**
- Can suggest more food or drink to admin
- Can manage all suggest

### CODE REVIEW

**Comment by Huan200799** [comment]
ready

**Comment by quoctd-0607** [comment]
commented
conflict code em

**Comment by Huan200799** [comment]
ready

File: routes/web.php [hunk 1]
```diff
@@ -37,3 +38,5 @@
         Route::get('delete/{id}', 'AdminUserController@delete')->name('users.delete');
     });
 });
+
+Route::resource('suggest', 'SuggestController');
```
Comment (quoctd-0607) [inline routes/web.php]: dùng only với các action dùng,tránh dư thừa resource

**Comment by Huan200799** [comment]
ready

**Pull request merged**